src/UserProfileService.Saga.Worker/SagaWorkerStartUp.cs

```diff
@@ -117,6 +118,8 @@ public override void RegisterMessaging(IServiceCollection serviceCollection, ICo
         serviceCollection.TryAddSingleton<ISagaCommandFactory, DefaultSagaCommandFactory>();
     }
 
+    
+    
```

Comment:
Leerzeilen

Reply:
I did not see your comment. I will change it later.